dist/bash-completion/tpm2-tools/tss2_decrypt

```diff
@@ -0,0 +1,24 @@
+# bash completion for tss2_decrypt -*- shell-script -*-
+
+_tss2_decrypt()
+{
+    local cur prev words cword split
+    _init_completion -s || return
+    case $prev in
+        -!(-*)h | --help)
+            COMPREPLY=( $(compgen -W "man no-man" -- "$cur") )
+            return;;
+        -!(-*)[kpc] | --keyPath | --plainText | --cipherText)
+            _filedir
```

Comment:
why does `--keyPath` here has the `_filedir` completion, but many other commands (e.g. tss2_authorizepolicy) don't? -> should be consistent.